components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/model/FilterQueryBuilder.java

```diff
@@ -32,6 +32,7 @@ public class FilterQueryBuilder {
     private List<String> timestampParameters = new ArrayList<>();
     private int count = 1;
     private String filter;
+    private boolean containSubAttribute;
```

Comment:
```suggestion
    private boolean isSubAttributeExist;
```

Reply:
or "hasSubAttribute"

Reply:
This is updated to `hasSubAttribute` and 
setter -> setHasSubAttribute
getter -> getHasSubAttribute